modules/edu.gemini.seqexec.server/src/main/scala/edu/gemini/seqexec/server/SeqTranslate.scala

```diff
@@ -70,7 +70,8 @@ class SeqTranslate(site: Site, systems: Systems, settings: Settings) {
 
     def notifyObserveStart: SeqAction[Unit] = otherSys.map(_.notifyObserveStart).sequenceU.map(_ => ())
 
-    def notifyObserveEnd: SeqAction[Unit] = otherSys.map(_.notifyObserveEnd).sequenceU.map(_ => ())
+    // endObserve must be sent to the instrument too.
+    def notifyObserveEnd: SeqAction[Unit] = (inst +: otherSys).map(_.notifyObserveEnd).sequenceU.map(_ => ())
```

Comment:
Why do we discard the result of `notifyObserveEnd`?

Reply:
Because it is a list of Units